### ISSUE TITLE
Missing alt text for letter images

### ISSUE THREAD

**Issue by wkdewey** [CDRH/family_letters#208]
In dealing with #205  I noticed that images in i.e. shan.L005 were missing alt text. The alt attribute is just "".
